apps/libexec/modules/merlin_db.py

```diff
@@ -28,9 +30,12 @@ def connect(mconf, reuse_conn=True):
 		try:
 			import MySQLdb as db
 		except:
-			print("Failed to import MySQLdb")
-			print("Install mysqldb-python or MySQLdb-python to make this command work")
-			sys.exit(1)
+			try:
+				import pymysql as db
+			except:
```

Comment:
Perhaps we should change both excepts to `except ImportError` instead? The blank `except:` catches even SIGINT etc.

Reply:
Agreed.